app/api/formations/[id]/route.ts

```diff
@@ -28,6 +28,8 @@ export async function PUT(
 
   await updateFormation(parseInt(id), data);
 
+  // clear the cache for the formation here
```

Comment:
is this a TODO?

Reply:
i had left this as a note for myself during some investigation, will remove. good catch!